tests/libpod/run.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package libpod
+
+import (
+	"context"
+
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	. "kubevirt.io/kubevirt/tests/framework/matcher"
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+func RunPodInNamespace(pod *k8sv1.Pod, namespace string) *k8sv1.Pod {
```

Comment:
It would be a good opportunity to follow up on this and:
- Just call it `Run` as it has the context of the package now. Unlike `Create`, it also waits to confirm running, that is why it is called differently.
- Require to input the namespace at the caller, do not assume it is part of the object (i.e. the current `RunPod` implementation).
- Replace asserts in the implementation with a returned error, allowing the caller to assert it and avoid the problem of ginkgo tracking the source of the error (i.e. to use of offset).